src/playermat/Playmat.ttslua

```diff
@@ -45,8 +45,9 @@ local RESOURCE_COUNTER
 
 -- global variable so it can be reset by the Clean Up Helper
 activeInvestigatorId = "00000"
+local drawButton = false
```

Comment:
Nit: showDrawButton

Reply:
That's already the name of the function, so not changing that

Reply:
Can always pick something else :)  drawButtonVisible, isDrawButtonVisible, etc.  The point about what the name implies this is holding still applies.